examples/large_models/gpt_fast/model_config.yaml

```diff
@@ -0,0 +1,11 @@
+#frontend settings
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 200
+responseTimeout: 300
+deviceType: "gpu"
+continuousBatching: false
+handler:
+    converted_ckpt_dir: "/home/ubuntu/serve/examples/large_models/gpt_fast/gpt-fast/checkpoints/meta-llama/Llama-2-7b-chat-hf/model.pth"
```

Comment:
Change default to int8 quantized `model_int8.pth`

Reply:
done